index.html

```diff
@@ -3,12 +3,28 @@
   <head>
     <title>dataverse-homepage-viz</title>
     <meta charset="utf-8" />
+    <link rel="stylesheet" href="css/bootstrap.min.css">
     <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
     <div id="circlepack"></div>
+    <div id="slider"></div>
+<!--    <div class="container">-->
+<!--      <div class="row">-->
+<!--        <div class="col">-->
+<!--            <p></p>-->
+<!--            <div id="circlepack"></div>-->
+<!--            <div id="slider"></div>-->
+<!--        </div>-->
+<!--      </div>-->
+<!--    </div>-->
```

Comment:
Added a Bootstrap container for testing. I will take out this commented code before merging it to the kw/feature-branch